ops/internal/report/l2.go

```diff
@@ -1,85 +1,41 @@
 package report
 
 import (
-	"context"
+	"errors"
 	"fmt"
+	"math/big"
 	"os"
 
 	"github.com/ethereum-optimism/optimism/op-chain-ops/foundry"
 	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
-	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/artifacts"
 	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/broadcaster"
-	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/inspect"
-	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm"
+	opcmv170 "github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/opcm/v170"
```

Comment:
I don't think I've seen versioning like this before. What makes this scheme preferable over more standard go module versioning where only the major version is included in the module name (e.g., `opcm/v1`)? Maybe this is necessary due to upstream contracts versioning, and the contracts are the one's that should be bumping major version more often?

Reply:
This maps to the contract version, not the Go version. OPD supports deploying multiple, backwards-incompatible versions of the contracts in one package so we need versioned sub-packages that contain the logic that describes how do deploy each version.

Changing the Go version to a new major version is a huge can of worms that I'd rather avoid while we're in fast development mode on OP Deployer.